charts/zac/README.md

```diff
@@ -19,6 +18,28 @@ A Helm chart for installing Zaakafhandelcomponent
 | @opentelemetry | opentelemetry-collector | 0.104.0 |
 | @solr | solr-operator | 0.8.1 |
 
+## Usage
```

Comment:
When / for what are these instructions needed? Do all developers need to do this now? Or only if you need to work in the Helm chart? What is the use case for this?

Reply:
Perhaps that's inspired by my questions around how to install this chart being a noob :) First thing I faced was that there are no repos specified for the charts we depend on.

Btw that's by no means complete guide as you'll also need: docker, kind (or another k8s cluster), kubectl, helm, helm-docs.